active/0000-control-transpilation.md

```diff
@@ -0,0 +1,99 @@
+- Start Date: 2016-12-07
+- RFC PR: (leave this empty)
+- Ember CLI Issue: (leave this empty)
+
+# Summary
+
+
+Allow consumer apps to control the transpilation config not only of application code, but also
+of addon code.
+
+# Motivation
+
+Ember-cli as allowed users to write ES6 code and transpile it to ES5/3 since almost the beginning
+of the project. It's one of those things that people take for granted.
+
+Today that is done through `ember-cli-babel`, an addon that is in the default blueprint and
+that the huge majority of apps use.
+
+This transpilation process has some sensitive defaults that work for most people, but also allow
+some use configuration for, by example, disable some transformations enabled by default or enable
+some experimental ones.
+
+What is less know is that this configuration only affects the application code. The transpilation
+of code living in addons is done according to the configuration of those addons. While there is
```

Comment:
to be specific: the app config affects the transpilation of everything inside `app/`, even those `app/` folders inside of addons!